### PR TITLE
removed all cosmos gateway

### PR DESCRIPTION
**Pull request by martin0995**
### Description

Removed all cosmos gateway related information from the docs. 

### Checklist

- [x] **Required** - I have added a label to this PR 🏷️
- [x] **Required** - I have run my changes through Grammarly
- [ ] If pages have been moved, I have created redirects in the `wormhole-mkdocs` repo

### CODE REVIEW

File: build/applications/wormhole-sdk.md [hunk 1]
```diff
@@ -279,7 +279,7 @@ The `WormholeTransfer` interface provides a convenient abstraction to encapsulat
 
 ### Token Transfers
 
-Performing a token transfer is trivial for any source and destination chains. You can create a new `Wormhole` object to make objects like `TokenTransfer`, `CircleTransfer`, and `GatewayTransfer`, to transfer tokens between chains.
+Performing a token transfer is trivial for any source and destination chains. You can create a new `Wormhole` object to make objects like `TokenTransfer`, and `CircleTransfer`, to transfer tokens between chains.
```
Comment (eshaben) [inline build/applications/wormhole-sdk.md:282]:
```suggestion
Performing a token transfer is trivial for any source and destination chains. You can create a new `Wormhole` object to make objects like `TokenTransfer` and `CircleTransfer`, to transfer tokens between chains.
```

File: learn/messaging/native-token-transfers/security.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ description: Explore the security measures of Native Token Transfers, including
 
 ## Global Accountant
 
-The Global Accountant is a defense-in-depth security feature that performs integrity checks on every token transfer. This feature isolates chain balances, ensuring more tokens cannot be burned and transferred out of a chain than were ever minted.
+The Global Accountant is a defense-in-depth security feature that checks the integrity of every token transfer. It ensures that chain balances remain isolated and more tokens cannot be burned and transferred out of a chain than were ever minted.
```
Comment (eshaben) [inline learn/messaging/native-token-transfers/security.md:10]: So the Global Accountant is not specific to Gateway?

**Review by eshaben — COMMENTED** [review]
I did a search for Gateway, looks like there are two more items that would need to be addressed:

- The `IBCRelayerAllowlistTemplate.md` snippet can be deleted
- The `build/contract-integrations/faqs.md` file uses references to contracts in the `gateway-integration` branch, these should probably be changed to `main` (double-check the line numbers)

File: learn/messaging/native-token-transfers/security.md (hunk 2 above)
Comment (martin0995) [reply learn/messaging/native-token-transfers/security.md:10]: I'm not sure. From the context and what I read about it, it looks like it's not. But I should confirm first. @ilariae ?

**Review by eshaben — APPROVED** [review]
LGTM. Created a ticket for the remaining stuff

**Pull request merged**